nipype/interfaces/base.py

```diff
@@ -881,7 +870,7 @@ def _outputs(self):
         """ Returns a bunch containing output fields for the class
         """
         outputs = None
-        if self.output_spec:
+        if self.output_spec is not None:
             outputs = self.output_spec()
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [self.output_spec is not callable](https://www.codacy.com/app/krzysztof-gorgolewski/nipy_nipype/file/1957504257/issues/source?bid=2767207&fileBranchId=2767718#l874)